Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -103,18 +114,39 @@ final class MessageBannerViewController: UIViewController, NibLoading {
       self.view.superview?.layoutIfNeeded()
     }
 
-    UIView.animate(withDuration: duration, delay: 0.0,
-                   options: UIView.AnimationOptions.curveEaseInOut,
-                   animations: { [weak self] in
-                    guard let self = self else { return }
-
-                    self.bottomConstraint?.constant = isHidden ? hiddenConstant : 0
-
-                    self.view.superview?.layoutIfNeeded()
-    }, completion: { [weak self] _ in
-      self?.view.isHidden = isHidden
-
-      self?.viewModel.inputs.bannerViewAnimationFinished(isHidden: isHidden)
+    UIView.animate(
+      withDuration: duration,
+      delay: 0.0,
+      options: UIView.AnimationOptions.curveEaseInOut,
```

Comment:
Can this infer `.curveEaseInOut` or `[.curveEaseInOut]`?

Reply:
Can try

Reply:
Hopefully it won't break the compiler :trollface: